inc/css/class-block-css.php

```diff
@@ -0,0 +1,322 @@
+<?php
+
+namespace ThemeIsle;
+
+if ( ! class_exists( '\ThemeIsle\BlockCSS' ) ) {
+	/**
+	 * Class BlockCSS
+	 *
+	 * @package BlockCSS
+	 */
+	class BlockCSS {
+
+		/**
+		 * The namespace under which the blocks are registered.
+		 *
+		 * @var string
+		 */
+		protected $library_prefix = 'themeisle-blocks';
+
+		/**
+		 * The namespace under which the block classees are saved.
+		 *
+		 * @var string
+		 */
+		protected static $blocks_classes = array();
+
+		/**
+		 * The namespace under which the fonts are saved.
+		 *
+		 * @var array
+		 */
+		protected static $google_fonts = array();
+
+		/**
+		 * BlockCSS constructor.
+		 *
+		 * @since   1.2.5
+		 * @access  public
+		 */
+		public function __construct() {
+			add_action( 'init', array( $this, 'autoload_block_classes' ), 99 );
+			add_filter( 'safe_style_css', array( $this, 'used_css_properties' ), 99 );
+		}
+
+		/**
+		 * Autoload classes for each block.
+		 *
+		 * @since   1.2.5
+		 * @access  public
+		 */
+		public function autoload_block_classes() {
+			$paths = glob( $this->get_dir() . '/../editor/*/*/*-css.php' );
```

Comment:
Can you implement a PSR-4 autoloader? Here you have the implementation logic 

https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-4-autoloader-examples.md#closure-example